server/pbench/bin/pbench-rsync-satellite

```diff
@@ -119,6 +123,12 @@ for host in $hosts ;do
         echo "FAILED:    rsync -av $remotehost:$remotearchive/$host/ ."
         let failures=failures+1
     fi
+    # move the symlink from TO-SYNC directory to SYNCED on the remote server.
+    ssh $remotehost "cd $remotearchive/$host; mv TO-SYNC/*.tar.xz SYNCED"
+
+    # remove the .prefix directory from remote server.
```

Comment:
The comment needs fixing: you are not removing the directory; you are removing all the files in it.